fog/sql_recovery_db/src/lib.rs

```diff
@@ -194,7 +195,13 @@ impl RecoveryDb for SqlRecoveryDb {
         &self,
         key: &CompressedRistrettoPublic,
         start_block: u64,
-    ) -> Result<bool, Error> {
+    ) -> Result<u64, Error> {
+        let highest_known_block_count: u64 = match self.get_highest_known_block_index() {
```

Comment:
this should be transactional with the insert operation i think

Reply:
Done.